fighters/ken/src/status.rs

```diff
@@ -187,3 +191,18 @@ unsafe extern "C" fn ken_attack_main_loop(fighter: &mut L2CFighterCommon) -> L2C
     }
     0.into()
 }
+
+// FIGHTER_STATUS_KIND_LANDING //
+
+#[status_script(agent = "ken", status = FIGHTER_STATUS_KIND_LANDING, condition = LUA_SCRIPT_STATUS_FUNC_INIT_STATUS)]
+pub unsafe fn landing_init(fighter: &mut L2CFighterCommon) -> L2CValue {
+    app::FighterSpecializer_Dolly::update_opponent_lr_1on1(fighter.module_accessor, *FIGHTER_STATUS_KIND_TURN_DASH);
+    common::shoto_status::fgc_init_landing(fighter);
+    original!(fighter)
+}
+
+#[status_script(agent = "ken", status = FIGHTER_STATUS_KIND_LANDING, condition = LUA_SCRIPT_STATUS_FUNC_EXEC_STATUS)]
+pub unsafe fn landing_exec(fighter: &mut L2CFighterCommon) -> L2CValue {
+    common::shoto_status::fgc_exec_landing(fighter);
```

Comment:
Any reason why Terry and Kazuya have the FighterSpecializer call but Ryu and Ken don't?

Reply:
added FighterSpecializer_Dolly call to Ryu/Ken (yes, it works)

Reply:
I figured it does, since Terry and Kazuya's unique 1 on 1 constant is the same value as Ryu's.